components/verifalia/actions/get-balance/get-balance.mjs

```diff
@@ -0,0 +1,23 @@
+import verifalia from "../../verifalia.app.mjs";
+
+export default {
+  name: "Get Credits Balance",
+  description: "Get the number of credit packs and free daily credits available to the account. " +
+        "[See the docs](https://verifalia.com/developers#credits-get-balance) for more information",
+  key: "verifalia-get-balance",
+  version: "0.0.2",
+  type: "action",
+  props: {
+    verifalia,
+  },
+
+  async run() {
+    const verifaliaClient = this.verifalia.buildVerifaliaRestClient();
+
+    return await this.verifalia.wrapVerifaliaApiInvocation(async () => {
```

Comment:
```suggestion
    return this.verifalia.wrapVerifaliaApiInvocation(async () => {
```

Reply:
@lcaresia This would just make the code more error prone, in my opinion - think what would happen if that code block would be later enclosed in a `try/catch`. For the sake of clarity, I prefer to leave this as is, thanks.